smoke_test/helpers/http_status_validation.rb

```diff
@@ -0,0 +1,16 @@
+module HttpStatusValidation
+module_function
+
+  def validate_response_status!
+    unless page.status_code == 200
```

Comment:
I think it would be better to check for 4xx or 5xx, as 2xx and 3xx responses are fine and redirects should be handled by Capybara.